client/src/js/app.js

```diff
@@ -373,6 +373,15 @@ function bhimaConfig($stateProvider, $urlRouterProvider, $urlMatcherFactoryProvi
     templateUrl: '/partials/suppliers/suppliers.html'
   })
 
+  /* purchase routes */
+
+
+  .state('purchasesCreate', {
+    url : '/purchases/new',
```

Comment:
We should formalise what we use for create paths - do you think we should use the `new` or `create` keyword?
@jniles @IMA-WorldHealth/local-contributors